DataFormats/Detectors/TPC/include/DataFormatsTPC/PIDResponse.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   PIDResponse.h
+/// @author Tuba Gündem, tuba.gundem@cern.ch
+///
+
+#ifndef AliceO2_TPC_PIDResponse_H
+#define AliceO2_TPC_PIDResponse_H
+
+// o2 includes
+#include "ReconstructionDataFormats/PID.h"
+#include "DataFormatsTPC/TrackTPC.h"
+
+namespace o2::tpc
+{
+
```

Comment:
I think TrackTPC can be forward declared:
```suggestion

namespace o2::tpc
{
  class TrackTPC;
```